galley/cmd/galley/cmd/server.go

```diff
@@ -140,9 +140,11 @@ func serverCmd() *cobra.Command {
 	svr.PersistentFlags().StringVar(&serverArgs.DomainSuffix, "domain", serverArgs.DomainSuffix,
 		"DNS domain suffix")
 	svr.PersistentFlags().BoolVar(&serverArgs.DisableResourceReadyCheck, "disableResourceReadyCheck", serverArgs.DisableResourceReadyCheck,
```

Comment:
These flags are still honored when the old processing pipeline is used. I marked them as deprecated proactively, as a way of documentation, so that people will no longer depend on them.

Reply:
When they are marked as deprecated with cobra, they are still honored. But they won't show up in the published docs.